js/main.js

```diff
@@ -0,0 +1,59 @@
+const getRandomInteger = function (num1, num2) {
+
+  if (typeof num1 !== 'number' || typeof num2 !== 'number') {
+
+    // Выход из функции при невалидном значении
```

Comment:
Лучше писать конкретнее что является невалидным значением, а то, что происходит выход из функции видно и без комментария
```suggestion
    // Аргументами функции могут быть только числа
```

Reply:
Можешь оставить как есть, но можно и бросить ошибку, это упростит работу с кодом в будущем. Есть альтернатива `console.error`, которую линтер пропустит:
```suggestion
    throw new Error('названиеФункции - что пошло не так конкретно');
```
Пока можешь особо не вникать в то, как это работает, в будущем это разберут на лекции